docs/quickstart-minikube.md

```diff
@@ -234,6 +234,17 @@ and binding it to our WordPress installation.
 helm install azure/wordpress --name quickstart
 ```
 
+Note: when installing the wordpress chart on some versions of Minikube, you
+may encounter issues due to [kubernetes/minikube#2256](https://github.com/kubernetes/minikube/issues/2256).
+If you're using
+[v0.24.1](https://github.com/kubernetes/minikube/releases/tag/v0.24.1), we recommend setting
+the `persistence.enabled` parameter to `false` using the following command.
+
+```console
+$ helm install --name my-release --namespace wp azure/wordpress \
+  --set persistence.enabled=false
+```
```

Comment:
The name and namespace here don't match what's written elsewhere in these instructions.

Should it be:

```
helm install azure/wordpress --name quickstart --set --set persistence.enabled=false
```

?